cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -59,13 +59,14 @@ import org.ossreviewtoolkit.utils.showStackTrace
 
 class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source code from a remote location.") {
     private val input by mutuallyExclusiveOptions(
-        option(
+        name = "Input Options",
```

Comment:
How about making the named `name` argument go last? That way, you could omit the `option1` and `option2` names, and the `name` argument would be closer to where usually the `inputGroup()` call is.

Reply:
Done, all of them.